packages/store/src/symbols.ts

```diff
@@ -62,3 +56,10 @@ export interface StoreOptions<T> {
    */
   children?: any[];
 }
+
+export interface ActionOptions {
+  /**
+   * Cancel the previous uncompleted request(s).
+   */
+  cancellable: boolean;
```

Comment:
what do you think of the "cancelPrevious"? it also avoids to British vs US spelling